tests/testthat/test-translate-access.r

```diff
@@ -0,0 +1,398 @@
+context("translate-ACCESS")
+
+# Access base_scalar conversions -----------------------------------------
+
+# Conversion
+
+test_that("as.numeric() translated to CDbl ", {
```

Comment:
Can you please reduce to one test per category? i.e. put all of the coercion functions in a single test.

And generally, I don't think it's worthwhile to test the straightforward translations.

Reply:
I removed most of the straightforward tests and kept only the special cases. Also reduced to 1 per category.